examples/mimas2/run_yosys.ys

```diff
@@ -0,0 +1,4 @@
+read_verilog example.v
+synth_xilinx -top example -family xc6s
+iopadmap -bits -outpad OBUF I:O -inpad IBUF O:I
+write_edif -pvector bra example.ndf
```

Comment:
Why `.ndf` file extension and not `.edif`?

Reply:
Good point, fixed.